src/main/java/com/opentok/MuteAllProperties.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * OpenTok Java SDK
+ * Copyright (C) 2021 Vonage.
+ * http://www.tokbox.com
+ *
+ * Licensed under The MIT License (MIT). See LICENSE file for more information.
+ */
+package com.opentok;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Defines values for the <code>properties</code> parameter of the
+ * {@link OpenTok#forceMuteAll(String, MuteAllProperties)} method.
+ *
+ * @see OpenTok#forceMuteAll(String, MuteAllProperties)
+ */
+public class MuteAllProperties {
+    private List<String> excludedStreams;
```

Comment:
Maybe this should be called `excludedStreamIds`. It is a list of stream IDs (strings) not Stream objects. (The Java SDK does include a Stream class.)